dashdotdb/services/serviceslometrics.py

```diff
@@ -149,13 +150,15 @@ def get_slometrics(self):
 
         return result
 
-    @staticmethod
+    @ staticmethod
```

Comment:
there's an unwanted whitespace here

Reply:
vscode decides it wants to add that space sometimes and I'm not sure why.  Fixed.